### PR TITLE
[doc-issue-14014] - Updated documentation regarding new endpoint.

### PR DESCRIPTION
**Pull request by GuiLeme**
Updated documentation regarding the `insertDocument` endpoint in the API

This documentation is linked with the issue [14014](https://github.com/bigbluebutton/bigbluebutton/pull/14264)

### CODE REVIEW

File: _data/insertDocument.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- name: "meetingID"
+  required: true
+  type: "String"
+  description: "A meeting ID that can be used to identify this meeting by the 3rd-party application. <br><br> This must be unique to the server that you are calling: different active meetings can not have the same meeting ID. <br><br> If you supply a non-unique meeting ID (a meeting is already in progress with the same meeting ID), then if the other parameters in the create call are identical, the create call will succeed (but will receive a warning message in the response). The create call is idempotent: calling multiple times does not have any side effect.  This enables a 3rd-party applications to avoid checking if the meeting is running and always call create before joining each user.<br><br> Meeting IDs should only contain upper/lower ASCII letters, numbers, dashes, or underscores.  A good choice for the meeting ID is to generate a <a href=\"http://en.wikipedia.org/wiki/Globally_unique_identifier\">GUID</a> value as this all but guarantees that different meetings will not have the same meetingID."
```
Comment (gustavotrott) [inline _data/insertDocument.yml:4]:
```suggestion
  description: "The meeting ID that identifies the meeting you want to insert documents."
```

File: _posts/2.5/2021-06-09-new.md [hunk 2]
```diff
@@ -67,7 +67,39 @@ Private guest lobby message, display position in waiting line
 
 A new API endpoint was added: `/insertDocument`. Now, it is possible to send a batch of documents with all the common presentation parameters such as `current`, `downloadable` and `removable`. 
 
-All the presentations are sent via payload (the body of the request), whether by a link to download the file or by `base64` encoded file.
+All the presentations are sent via payload (the body of the request) such as when preuploading documents in `/create` endpoint, whether by a link to download the file or by `base64` encoded file.
```
Comment (antobinary) [inline _posts/2.5/2021-06-09-new.md:70]: Consider just linking to the api page section insertDocument, so you don't need to copy the same info twice

File: _posts/dev/2015-04-05-api.md [hunk 3]
```diff
@@ -79,6 +80,11 @@ Updated in 2.5:
 
 - **create** - **Added:** `meetingCameraCap`, `groups`, `disabledFeatures`, `meetingExpireIfNoUserJoinedInMinutes`, `meetingExpireWhenLastUserLeftInMinutes`; **Deprecated:** `learningDashboardEnabled`, `breakoutRoomsEnabled`, `virtualBackgroundsDisabled`.
 
+- **insertDocument**
```
Comment (antobinary) [inline _posts/dev/2015-04-05-api.md:83]: `- **insertDocument** endopoint was first introduced` should suffice here. It's just a list of api changes per BBB release for quick reference

File: _posts/dev/2015-04-05-api.md [hunk 4]
```diff
@@ -529,6 +536,54 @@ There is only an XML response for this call if the `redirect` parameter is set t
 </response>
 ```
 
+## insertDocument
```
Comment (antobinary) [inline _posts/dev/2015-04-05-api.md:539]: Nice! I love the granularity!

Comment (GuiLeme) [reply _posts/dev/2015-04-05-api.md:536]: Thanks :smile:

File: _posts/dev/2015-04-05-api.md [hunk 5]
```diff
@@ -529,6 +533,54 @@ There is only an XML response for this call if the `redirect` parameter is set t
...
+
+This endpoint insert one or more documents into a running meeting via API call
+
+**Resource URL:**
+
+http&#58;//yourserver.com/bigbluebutton/api/insertDocument?[parameters]&checksum=[checksum]
```
Comment (antobinary) [inline _posts/dev/2015-04-05-api.md:542]:
```suggestion
https&#58;//yourserver.com/bigbluebutton/api/insertDocument?[parameters]&checksum=[checksum]
```

File: _posts/2.5/2021-06-09-new.md [hunk 6]
```diff
@@ -61,6 +61,14 @@ You can now pin a webcam so it always stays visible.  This is useful if one on o
 
 Private guest lobby message, display position in waiting line
 
+### Improvements to Endpoint API 
+
+#### New endpoint
+
+A new API endpoint was added: `/insertDocument`. Now, it is possible to send a batch of documents with all the common presentation parameters such as `current`, `downloadable` and `removable`. 
+
+All the presentations are sent via payload (the body of the request) such as when preuploading documents in `/create` endpoint, whether by a link to download the file or by `base64` encoded file. See the [API documentation](../dev/api.html#insertdocument) for more information
```
Comment (antobinary) [inline _posts/2.5/2021-06-09-new.md:70]:
```suggestion
All the presentations are sent via payload (the body of the request) such as when preuploading documents in `/create` endpoint, whether by a link to download the file or by `base64` encoded file. See the [API documentation](/dev/api.html#insertdocument) for more information
```

or full link (I think jekyll will have troubles with `../` but I have not actually ran it, just proofreading

**Review by antobinary — APPROVED** [review]

**Review by gustavotrott — APPROVED** [review]

**Pull request merged**